xbzu1/1.0/preset.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no" ?> 
+<!DOCTYPE ip_presets SYSTEM "C:/Xilinx/Vivado/2021.1/data/boards/board_schemas/current/preset.dtd">
```

Comment:
Vivado will complain, but this line isn't needed. In this case the .dtd file will only be found if the user is using Windows and Vivado is installed in the default path.

Reply:
Will remove after done with debug.

Reply:
We should be able to use an env var to substitute the Vivado path.  Like this:
"{$XILINX_VIVADO}/data/boards/board_schemas/current/preset.dtd"

Reply:
I think it either has to be removed or commented out. Would you prefer to use an env var AND then also comment it out? Or just remove it when I am done debugging?